assets/js/blocks/cart-checkout/checkout/block.js

```diff
@@ -23,6 +23,8 @@ import FormattedMonetaryAmount from '@woocommerce/base-components/formatted-mone
 import {
 	CheckoutProvider,
 	useValidationContext,
+	ValidationContextProvider,
```

Comment:
This is a rebase conflict resolution error. In Albert's recent pull `ValidationContextProvider` was moved to `blocks/cart-checkout/checkout/frontend.js`